libs/blocks/bulk-publish-v2/components/bulk-publisher.js

```diff
@@ -0,0 +1,419 @@
+            @click=${this.submit}>
+            Run Job
+            <span class="loader${this.processing === 'started' ? '' : ' hide'}"></span>
+          </button>
+        </div>
+        <label class="process-title" for="Urls">
+          <strong>PAGE URLs</strong>
+        </label>
+      </div>
+      <div class="urls${typeof this.disabled !== 'boolean' ? ' invalid' : ''}">
+        <div class="error-bar">${this.renderErrorBar()}</div>
+        <div class="checkmark${this.disabled ? '' : ' show'}">
+          <img src=${checkmarkIcon} alt="Valid Urls" />
+        </div>
+        <div class="entered-count${this.urls.length ? ' show' : ''}">${this.urls.length}</div>
+        <textarea 
+          id="Urls"
+          wrap="off"
+          placeholder="Example: https://main--milo--adobecom.hlx.page/path/to/page"
+          @blur=${this.setUrls}
+          @change=${this.setUrls}></textarea>
+      </div>
+    `;
+  }
+
+  getJobState() {
+    const state = {
+      showList: this.mode === 'half' || this.openJobs,
+      showClear: this.jobs.length && this.processing === false,
+      loading: this.processing !== false,
+    };
+    Object.keys(state).forEach((key) => (state[key] = `${state[key] ? '' : ' hide'}`));
+    state.count = this.jobs.reduce((count, { result }) => {
+      const paths = result?.job?.data?.paths?.length ?? 0;
+      return count + paths;
+    }, 0);
+    return state;
+  }
+
+  setJobStopped(event) {
+    const status = event.detail;
+    const jobProcess = this.jobs.find(({ result }) => result.job.name === status.name);
+    if (jobProcess) jobProcess.status = status;
+    if (this.jobs.filter((job) => !job.status).length === 0) {
+      this.processing = false;
+      sticky().set('resume', []);
+    }
+  }
+
+  setJobProgress(event) {
+    const { name, progress } = event.detail;
+    const jobProcess = this.jobs.find(({ result }) => result.job.name === name);
+    if (jobProcess) jobProcess.progress = progress;
+    this.requestUpdate();
+  }
+
+  renderProgress(done) {
+    if (!done) return '';
+    return `${this.jobs.reduce((count, { progress }) => {
+      const processed = progress?.processed ?? 0;
+      return count + processed;
+    }, 0)}/${done}`;
+  }
+
+  clearJobs = () => {
+    this.jobs = [];
+    if (this.mode === 'full') this.openJobs = false;
+  };
+
+  renderResults() {
+    const { showList, showClear, loading, count } = this.getJobState();
+    const handleToggle = () => {
+      if (!this.openJobs) {
+        this.openJobs = !!this.jobs.length;
+      }
+    };
+    return html`
+      <div
+        class="panel-title"
+        @click=${handleToggle}>
+        <span class="title">
+          ${count ? html`<strong>${count}</strong>` : ''}
+          Job Results
+        </span>
+        <div class="jobs-tools${showList}">
+          <div 
+            class="clear-jobs${showClear}"
+            @click=${this.clearJobs}>
+            <img src=${clearJobsIcon} alt="Clear Job List" />
```

Comment:
```suggestion
            <img src=${clearJobsIcon} alt="Clear list icon" title="Clear Job List" />
```
nit: accessibility - Title attribute for tooltip on hover.